dbt_subprojects/dex/models/automated_trades/dex_automated_base_trades.sql

```diff
@@ -1,18 +1,17 @@
 {{ config(
     schema = 'dex'
     , alias = 'automated_base_trades'
-    , partition_by = ['block_month', 'blockchain', 'project']
+    , partition_by = ['block_month', 'blockchain']
```

Comment:
should we partition here by `block_date` instead, to match the final table? why differ between the two?
if so, we should apply `block_date` to unique key